test/node/api-versions.js

```diff
@@ -228,3 +217,34 @@ test.test('must throw an error on invalid version in application parameter',
     test.end();
   }
 );
+
+test.test('must set version to (1.0.0) if none was provided', (test) => {
+  const serverConfig = {
+    applications: [appWithoutVersion], authPolicy: app.authCallback,
+  };
+  server = jstp.net.createServer(serverConfig);
+  server.listen(0, () => {
+    const port = server.address().port;
+    const appV1Compatible = { name: app.name, version: '1.0.0' };
+    // must connect to appWithoutVersion
+    jstp.net.connect(appV1Compatible, null, port, (error, conn) => {
+      connection = conn;
+      test.assertNot(error, 'connect must not return an error');
+      connection.callMethod('calculator', 'answer', [], (error, result) => {
+        test.assertNot(error, 'callMethod must not return an error');
+        test.strictSame(result, 13);
+        test.end();
+      });
+    });
+  });
+});
+
```

Comment:
I think we should also add test that'll test 'must connect to app without version('latest') when client didn't specify version'

Reply:
@lundibundi there is no app without version now, and we already have a test that checks if a user connects to the latest app, if no version was specified.